storybookjs/storybook
# UI: Boolean control change for UI Fix toggle button

Issue: [UI: Fix toggle button for custom theming #13924](https://github.com/storybookjs/storybook/issues/13924)

**Boolean control change for UI Fix toggle button for custom themeing**

_Apologies in advance if anything here is out of your norm.. First time contributing to an O/S project + first time using storybook... 😟_

## What I did
**Reproduced by:** git cloning, added custom theming [ .storybook/YourTheme.js (complex example) ](https://storybook.js.org/docs/react/configure/theming) and a control button as per examples on playbook website.

When adding the custom theme it seems Control takes the property form **barBg** as indicated beforehand, the propose change will make it take from **appBg** instead.

Example of my custom theme 

```
    // UI
    appBg: 'white',
    appContentBg: 'silver',
    appBorderColor: 'grey',
    appBorderRadius: 4,  

    // Toolbar default and active colors
    barTextColor: 'silver',
    barSelectedColor: 'black',
    barBg: 'black',
  
```

## How to test
**Tested by:**  Reproducing above and then applying fix to Boolean.tsx and building locally with **yarn build-storybook** to test (Sorry no automation this time...)

![playbook](https://user-images.githubusercontent.com/18439866/113102160-5f0eb780-9249-11eb-8f1f-a1f66746ac2a.PNG)

- Is this testable with Jest or Chromatic screenshots?
Not sure, sorry.
- Does this need a new example in the kitchen sink apps?
Not that I'm aware of.
- Does this need an update to the documentation?
Not that I'm aware of.

If your answer is yes to any of these, please make sure to include it in your PR.
